dp/generate_climos.py

```diff
@@ -366,10 +370,29 @@ def update_metadata_and_time_var(input_file, t_start, t_end, climo_filepath):
             raise ValueError('Expected climo file to contain # time values in {}, but found {}'
                              .format(num_times_to_interval_set.keys(), cf.time_var.size))
 
+
+        # Update cell_methods to reflect the operation being done to the data
+        cell_method_op = ''
+        if operation == 'std':
+            cell_method_op = 'standard_deviation'
+        elif operation == 'mean':
+            cell_method_op = operation
```

Comment:
This is harder than necessary to understand, and without validation of the value of `operation` can result in an erroneous value for `cell_method_op`. 

Assuming `operation` is validated, the following code is clearer and more concise. Generally speaking, using hashes to map one set of values to another is the clearest and cleanest tool for the job, and should be used wherever feasible. 

```python
cell_method_op = {
   'std': 'standard_deviation',
   'mean': 'mean',
}[operation];
```

Reply:
Agreed!